docs/content/2.components/CldMediaLibrary.md

```diff
@@ -0,0 +1,94 @@
+# CldMediaGallery.vue
```

Comment:
thoughts on using the name CldMediaLibrary to reflect the actual widget? https://cloudinary.com/documentation/media_library_widget

Reply:
Yes good idea! I must have made a typo there :)